app/src/main/java/com/dimelo/sampleapp/RcSourceModel.java

```diff
@@ -0,0 +1,74 @@
+package com.dimelo.sampleapp;
+
+import android.content.Context;
+import android.text.TextUtils;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.reflect.TypeToken;
+
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+
+public class RcSourceModel {
+    String domainName;
```

Comment:
```suggestion
    private String domainName;
```

same thing for others

Reply:
if I put this Private I cannot read this from another file